axelor-sale/src/main/java/com/axelor/apps/sale/service/saleorder/SaleOrderLineServiceImpl.java

```diff
@@ -917,7 +928,8 @@ public void fillPriceFromPackLine(SaleOrderLine saleOrderLine, SaleOrder saleOrd
       if (!saleOrderLine.getEnableFreezeFields()) {
         saleOrderLine.setPrice(exTaxPrice);
         saleOrderLine.setInTaxPrice(
-            taxService.convertUnitPrice(false, saleOrderLine.getTaxLine(), exTaxPrice));
+            taxService.convertUnitPrice(
+                false, saleOrderLine.getTaxLine(), exTaxPrice, AppBaseService.COMPUTATION_SCALING));
```

Comment:
it looks like it is directly set in the `inTaxPrice` field in sale order line here, so please use the scale for unit price.